docs/examples/adr-pvarray/plot_simulate_fast.py

```diff
@@ -0,0 +1,171 @@
+"""
+Fast simulation using the ADR efficiency model with PVsyst parameters
```

Comment:
I would prefer "Fast approximation of PVsyst single diode model using the ADR efficiency model"

Reply:
Not sure about this one.  It's really about simulation, i.e. running a time series.

Reply:
It's a minor point, someone might read that title and assume that the ADR model takes PVsyst parameters as input. Maybe "starting from PVsyst parameters."

Reply:
That works.